packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -103,6 +103,15 @@ enum HostArtifact {
   pubExecutable,
 }
 
+// TODO(knopp): Remove once darwin artifacts are moved out of darwin-x64
+String _platformDirName(TargetPlatform platform) {
+  if (platform == TargetPlatform.darwin) {
+    return 'darwin-x64';
+  } else {
```

Comment:
nit: no else needed since its a return above:

```dart
if (platform == TargetPlatform.darwin) {
  return 'darwin-x64';
}
return getNameForTargetPlatform(platform);
```